integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -7355,6 +7368,11 @@ msgstr ""
 "Dieser Ort kann nicht in der Web-App geöffnet werden, weil er nicht "
 "veröffentlicht ist."
 
+#: cms/templates/pois/poi_list_row.html
+msgid "You cannot archive a location which is used by an event."
+msgstr ""
+"Der Ort kann nicht archiviert werden, da er von einem Event verwendet wird."
```

Comment:
```suggestion
"Der Ort kann nicht archiviert werden, da er von einer Veranstaltung verwendet wird."
```
Was there a reason why you chose to stick with "einem Event" or did you just forgot/oversee it? :)

Maybe a bit of background for my reasoning: While it's certainly not unusual to use Event also as a German word and municipalities would definitely understand, I think staying consistent within our system is important (and we usually use "Veranstaltung") :)

Reply:
Nö. no special meaning 😅